### PR TITLE
Add CommandAction function

### PR DESCRIPTION
**Pull request by jereksel**
When you just want to set Action when using Command

### CODE REVIEW

**Comment by jawher** [comment]
Thanks @jereksel for your contribution !

To keep the surface area as small as possible, and to avoid code duplication, what do you think about implementing this differently: instead of exposing a `CommandAction` function, provide instead a (top-level) function which takes the action function and returns one suitable for a command initializer ?
Concretely, it would look like this:


```go
app.Command("list", "list all configs", cli.ActionCommand(func()) {
  // ...
}))
```

**Comment by jereksel** [comment]
Sure, sound good to me

File: README.md [hunk 1]
```diff
@@ -257,6 +257,18 @@ bzk.Command("job", "actions on jobs", func(cmd *cli.Cmd) {
     cmd.Command("log", "show a job log", nil)
 })
 ```
+When you just want to set Action to cmd, you can use ActionCommand function for this
+```go
+app.Command("list", "list all configs", ActionCommand(func() { list() }))
```
Comment (jawher) [inline README.md]: `ActionCommand` needs to be prefixed with the package name `cli`.

Also, this should also work:

```go
app.Command("list", "list all configs", cli.ActionCommand(list))
```

**Comment by jawher** [comment]
@jereksel Thanks for the quick fix.
One last remark to handle and it should be good for mege 👍

**Comment by jawher** [comment]
@jereksel Thank you for your contribution !

**Pull request merged**